bears/stylus/StylintBear.py

```diff
@@ -0,0 +1,36 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='stylint',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+)[:]*(?P<column>\d+)*\s*.*'
+                     r'(?P<severity>error|warning)\s*(?P<message>.+)')
+class StylintBear:
+    """
+    Attempts to catch little mistakes (duplication of rules for instance) and
+    to enforce a code style guide. This is particularly important with Stylus,
```

Comment:
Actually I don't know Stylus. Maybe you could make a small line explaining what it is? Very shortly, maybe like
```
This is particularly important with Stylus (a common XYZ thing), ...
```